exec/java-exec/src/main/java/org/apache/drill/exec/rpc/user/UserSession.java

```diff
@@ -115,6 +118,7 @@ public UserSession build() {
 
   private UserSession() {
     queryCount = new AtomicInteger(0);
+    sessionId = nextSessionId.getAndIncrement();
   }
```

Comment:
Sorry, I see it's static. But using this approach session id on different drillbits may overlap.